C++/UniquePaths||.cpp

```diff
@@ -0,0 +1,56 @@
+class Solution {
```

Comment:
/*
Problem Satement:-
A robot is located at the top-left corner of a m x n grid 
The robot can only move either down or right at any point in time. The robot is trying to reach the bottom-right corner of the grid (marked 'Finish' in the diagram below).
Now consider if some obstacles are added to the grids. How many unique paths would there be?
An obstacle and empty space is marked as 1 and 0 respectively in the grid.
The solution uses Dynamic approach to find the total number of unique paths for the robot to reach end point (n-1,m-1) from start point(0,0)
*/